src/components/AddItem/AddItem.js

```diff
@@ -1,27 +1,108 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { db } from '../../lib/firebase';
-import { collection, addDoc } from 'firebase/firestore';
+import { collection, addDoc, serverTimestamp } from 'firebase/firestore';
 import Navigation from '../Navigation/Navigation';
 
 const AddItem = () => {
+  const [itemName, setItemName] = useState('');
+  const [purchaseFreq, setPurchaseFreq] = useState('');
+  const [lastPurchaseDate, setLastPurchaseDate] = useState(null);
+  // const date = new Date();
+
   async function handleClick(e) {
-    e.preventDefault();
+    await setItemName(e.target.value);
+    await setPurchaseFreq(e.target.value);
+    await setLastPurchaseDate(e.target.value);
+
+    // const purchaseDate = lastPurchaseDate.setDate(lastPurchaseDate.getDate() + {purchaseFreq});
+    // console.log("This is the next purchase date", purchaseDate.toDateString());
 
     try {
       const colRef = collection(db, 'users');
       const docRef = await addDoc(colRef, {
-        item: 'bread',
+        item: { itemName },
+        purchase: { purchaseFreq },
+        createdAt: serverTimestamp(),
+        purchaseDate: { lastPurchaseDate },
       });
       console.log('Document written with ID: ', docRef.id);
+      console.log('Saved state of frequenxy', purchaseFreq);
     } catch (e) {
       console.error('Error adding document: ', e);
     }
   }
 
+  // function handleFreqChange(e){
+  //   setPurchaseFreq(e.target.value);
+  // }
+
+  //for last purchase date
+  //hypothetically, we could do a created at stamp. In which, whenever the item was added to the list, that data is then "created" at that day and time.
+  //so dynamically, we can add +7, +14, +30 to the day that it was created
+
   return (
     <>
       <Navigation />
       <h1>Smart Shopping List</h1>
+      <div>
+        <form>
+          <label>
+            <input
+              type="text"
+              required
+              value={itemName}
+              onChange={(e) => setItemName(e.target.value)}
+            ></input>
+            Name
+          </label>
+        </form>
+        <form>
+          <div className="radio">
```

Comment:
I agree, labels on the radio buttons are necessary for screen readers so we should add ARIA attributes. I found this nice guide specifically about ARIA with radio buttons -- https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20160317/examples/radio/radio.html 

I think @timothy-taylor is right that we could leverage event bubbling in React for the onChange event, but it's more of a stylistic change so that's up to y'all!